packages/notifier/src/notifier.js

```diff
@@ -138,8 +139,8 @@ export const makeNotifierKit = (...args) => {
     },
   });
 
-  if (args.length >= 1) {
-    updater.updateState(args[0]);
+  if (initialState !== deferInitialState) {
+    updater.updateState(deferInitialState);
```

Comment:
Found it. 

```suggestion
    updater.updateState(initialState);
```

This should have damaged almost any use of notifiers. I'm alarmed that only two CI tests failed.

Reply:
I should have run tests before requesting review. When I fixed this error, all tests passed. So I added a test for what should have failed with the default param change: [121e248](https://github.com/Agoric/agoric-sdk/pull/5255/commits/121e24813703f203fc8ae76d579709cf0e123055)